pkg/controller/registry/reconciler/grpc.go

```diff
@@ -141,6 +156,9 @@ func (c *GrpcRegistryReconciler) EnsureRegistryServer(catalogSource *v1alpha1.Ca
 		return errors.Wrapf(err, "error ensuring pod: %s", source.Pod().GetName())
 	}
 	if err := c.ensureUpdatePod(source); err != nil {
+		if _, ok := err.(UpdateNotReadyErr); ok {
```

Comment:
nit: could use errors.Is

Reply:
nice, forgot about this func. much cleaner. will use it next time